src/views/clusteroverview/SettingsTab/ClusterTab/components/PersistentReservationSection/persistent-reservation-section.scss

```diff
@@ -0,0 +1,5 @@
+.persistent-reservation-section--loading {
+  :hover {
+    cursor: wait;
+  }
+}
```

Comment:
can we do a kubevirt global class for it? so if we want to have other switches with this loading we don't have to create other classes ?